graph/adjacency/impl/ThingAdjacencyImpl.java

```diff
@@ -95,7 +95,7 @@
             iids = newIIDs;
         }
 
-        return iterate(iids).flatMap(iid -> iterate(edges.get(iid).values()));
+        return iterate(iids).flatMap(iid -> iterate(edges.get(iid) != null ? edges.get(iid).values().iterator() : emptyIterator()));
```

Comment:
Added a null check so that we don't falsely assume edges already exist while it's still being loaded. In case of the edges are null, returning an empty iterator is safe since it will be merged together with persisted edges that contain every edge.